### PR TITLE
Configure auto merge

### PR DESCRIPTION
**Pull request by ptbrowne**
I want to add this for very small PRs : https://github.com/bobvanderlinden/probot-auto-merge

The idea being that for very small PRs, we can label when creating the issue with "merge" and
as soon as it is reviewed, it is automatically merged.

### CODE REVIEW

File: auto-merge.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+min-approvals:
+  COLLABORATOR: 1
+requireLabes:
```
Comment (Crash--) [inline auto-merge.yml]:
```suggestion
requiredLabels
```

File: auto-merge.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+- merg
```
Comment (Crash--) [inline auto-merge.yml]:
```suggestion
- merge
```

**Comment by Crash--** [comment]
I like the idea! 

I also like the `deleteBranchAfterMerge` option!

**Comment by ptbrowne** [comment]
Thanks for the fixes Quentin 😅

> `deleteAfterMerge` option

I already added https://probot.github.io/apps/delete-merged-branch/ so I think it will do the job :)

File: auto-merge.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+min-approvals:
+  COLLABORATOR: 1
```
Comment (Crash--) [inline auto-merge.yml]: I would be more confortable with `member`

File: auto-merge.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+requiredLabels
```
Comment (Crash--) [inline auto-merge.yml]:
```suggestion
requiredLabels:
```

sorry :'(

File: auto-merge.yml (hunk 3 above)
Comment (ptbrowne) [reply auto-merge.yml]: Done, you're right, it's better.

File: auto-merge.yml (hunk 4 above)
Comment (ptbrowne) [reply auto-merge.yml]: My bad.. what is wrong with me !

**Pull request merged**